PatientRegistrationApp/src/components/SinglePatientHomeAddress.vue

```diff
@@ -96,30 +99,29 @@ import EventBus from '../eventBus'
 	data () {
 		return {
 			postalCodeRules: [
-        (v) => !!v || "Zip code is required",
-			(v) =>
-			/(^\d{5}$)|(^\d{5}-\d{4}$)/.test(v) ||
-			"Zip code must be in format of ##### or #####-####",
-		],
-			
-			state:[
-		'AL', 'AK', 'AS', 'AZ',
-		'AR', 'CA', 'CO', 'CT',
-		'DE', 'DC', 'FM',
-		'FL', 'GA', 'GU', 'HI', 'ID',
-		'IL', 'IN', 'IA', 'KS', 'KY',
-		'LA', 'ME', 'MH', 'MD',
-		'MA', 'MI', 'MN', 'MS',
-		'MO', 'MT', 'NE', 'NV',
-		'NH', 'NJ', 'NM', 'NY',
-		'NC', 'ND', 'MP', 'OH',
-		'OK', 'OR', 'PW', 'PA', 'PR',
-		'RI', 'SC', 'SD', 'TN',
-		'TX', 'UT', 'VT', 'VI', 'VA',
-		'WA', 'WV', 'WI', 'WY',
-		],
+				(v) => !!v || "Zip code is required",
+				(v) =>
+				/(^\d{5}$)|(^\d{5}-\d{4}$)/.test(v) ||
+				"Zip code must be in format of ##### or #####-####",
```

Comment:
Let's make a card to restrict the inputs on zipcode to be one of these two formats. We'll probably prioritize this for later, but we should be consistent with our checks.

Reply:
Created MVS-537 to handle this

Reply:
👍